sorl_watermarker/engines/base.py

```diff
@@ -66,7 +67,15 @@ def watermark(self, image, options):
             raise AttributeError('Trying to apply a watermark, '
                                  'however no THUMBNAIL_WATERMARK defined, '
                                  'and watermark not set on tag')
+        # Note - I do not think find can be used for this purpose - it is used for collecting static files to the static folder
+        # Use os.path.join with the static root instead, and if not found, raise the error.
         watermark_path = find(watermark_img)
+        if not watermark_path:
+            watermark_path = os.path.join(settings.STATIC_ROOT, watermark_img)
+            if not os.path.isfile(watermark_path):
+                raise AttributeError('Trying to apply a watermark, '
+                                     'however no THUMBNAIL_WATERMARK defined, '
+                                     'or watermark can not be found!')
```

Comment:
Could you please update the error messages (the current one and the one above #67, so it will be shorted and describe the error more clearly?